qiskit/aqua/operators/primitive_ops/primitive_op.py

```diff
@@ -26,6 +26,9 @@
 from ..operator_base import OperatorBase
 from ..legacy.base_operator import LegacyBaseOperator
 
+if TYPE_CHECKING:
+    from .matrix_op import MatrixOp
+
```

Comment:
I don't see `MatrixOp` or `TYPE_CHECKING` be used in these changes, what are they used for?

Reply:
Thank you. I forgot to change the code. I would like to make type of `to_matrix_op` Union[MatrixOp, ListOp].